src/Middleware/Diagnostics/test/UnitTests/DeveloperExceptionPageMiddlewareTest.cs

```diff
@@ -540,8 +540,8 @@ public async Task UnhandledError_ExceptionNameTagAdded()
     {
         // Arrange
         var meterFactory = new TestMeterFactory();
-        using var requestDurationCollector = new MetricCollector<double>(meterFactory, "Microsoft.AspNetCore.Hosting", "http-server-request-duration");
-        using var requestExceptionCollector = new MetricCollector<long>(meterFactory, DiagnosticsMetrics.MeterName, "diagnostics-handler-exception");
+        using var requestDurationCollector = new MetricCollector<double>(meterFactory, "Microsoft.AspNetCore.Hosting", "http.server.duration");
+        using var requestExceptionCollector = new MetricCollector<long>(meterFactory, DiagnosticsMetrics.MeterName, "aspnet.diagnostics_handler.exceptions");
```

Comment:
```suggestion
        using var requestExceptionCollector = new MetricCollector<long>(meterFactory, DiagnosticsMetrics.MeterName, "aspnet.diagnostics.exceptions");
```

(following your suggestion on https://github.com/lmolkova/semantic-conventions/pull/1)